ipatests/test_integration/test_idp.py

```diff
@@ -58,11 +61,11 @@ def kinit_idp(host, user, keycloak_server):
     tasks.kdestroy_all(host)
     # create armor for FAST
     host.run_command(["kinit", "-n", "-c", ARMOR])
-    since = time.strftime('%Y-%m-%d %H:%M:%S')
     cmd = ["kinit", "-T", ARMOR, user]
     with host.spawn_expect(cmd, default_timeout=100) as e:
         e.expect('Authenticate at .+: ')
-        uri = get_verification_uri(host, since, keycloak_server.hostname)
+        time.sleep(15)
```

Comment:
Why this change? Was the test unstable?

Reply:
Its stable in PR-CI environment but its unstable in idm-ci environment machines are a bit faster.